contentcuration/contentcuration/views.py

```diff
@@ -252,6 +252,8 @@ def _duplicate_node(node, sort_order=1, parent=None):
         changed=True,
         original_node=node.original_node or node,
         cloned_source=node,
+        original_channel_id = node.original_channel_id or node.original_node.get_channel().id,
+        source_channel_id = node.get_channel().id,
```

Comment:
Niiiiice! Clever way to do it. Will this cover the bases for clipboard-based cloning as well?